ctl/install/install.go

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package install
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+
+	"kmesh.net/kmesh/ctl/utils"
+)
+
+type GitHubFile struct {
+	Name        string `json:"name"`
+	DownloadURL string `json:"download_url"`
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "install",
+		Short: "install kmesh with all the resources",
+		Example: `# Install all kmesh resources (defaults to main):
+kmeshctl install
+
+# Install a specific version of kmesh
+kmeshctl install --version 0.5`,
+		Run: func(cmd *cobra.Command, args []string) {
+			version, err := cmd.Flags().GetString("version")
```

Comment:
We still have issue #949.
@YaoZengzeng  Has this problem been fixed?

Reply:
No, I hope this issue could be fixed with the solution of #892, but it seems that this problem is not being addressed now.  If no one has dealt it by then, I will fix it.